src/i18n/en/Payroll.PayrollOverview.json

```diff
@@ -1,28 +1,52 @@
 {
-  "pageTitle": "Review payroll for {{startDate}} - {{endDate}}",
-  "buttons": {
-    "edit": "Edit",
-    "editTitle": "Edit",
-    "submit": "Submit",
-    "submitTitle": "Submit"
-  },
+  "pageTitle": "Review payroll",
+  "pageSubtitle": "Regular payroll for {{startDate}} - {{endDate}}",
+  "submitCta": "Submit",
+  "editCta": "Edit payroll",
+
   "alerts": {
+    "payrollNotCalculated": "Payroll is not calculated",
     "progressSaved": "Your progress has been saved",
     "directDepositDeadline": "To pay your employees with direct deposit on {{payDate}}, you'll need to run payroll by {{deadline}}",
     "missedDeadlineWarning": "If you miss this deadline, your employees' direct deposit will be delayed."
   },
-  "sections": {
-    "payrollSummary": "Payroll Summary"
-  },
+  "payrollSummaryTitle": "Payroll Summary",
+  "payrollSummaryLabel": "Payroll summary table",
   "dataViews": {
-    "summary": "Summary",
-    "configuration": "Configuration"
+    "label": "Payroll details",
+    "companyPaysTab": "Company pays details",
```

Comment:
Looks like this is just "Company pays"?

https://www.figma.com/design/6dxOSiONDiJoa9zY1wOwbs/Frontend-SDK-Partner-Design-File?node-id=9604-28976&t=n8Ys6vSxrlJ3BUOp-4